src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom"
+
+const ArticleCard = (props) => {
+    const article = props.article
+    console.log(article)
+    return (
+        <>
+        <div className="article-card-wrapper">
+            <Link to={`/articles/${article.article_id}`} className="article-card-link">
+            <div className="article-card-image-and-title">
+                <img className="article-card-image" src={article.article_img_url} alt="a thumbnail image representing this article" />
```

Comment:
how can we make the alt description more specific to the image? at the moment all image articles have the same alt description which isnt very accessible